.eslintignore

```diff
@@ -2,3 +2,4 @@ test/*.conf.js
 node_modules
 test/mock-server/**/*.js
 coverage
+src/components/Auth/assets/animations.scss
```

Comment:
You can run `prettier --write src/components/Auth/assets/animations.scss` to fix the lint errors

Reply:
I've tried both methods to prettify the file but somehow I'm gettings this message even after running the file through prettier and format:
```
src/components/Auth/assets/animations.scss
 21:13  ✖  Unexpected whitespace after "("    function-parentheses-space-inside
 23:4   ✖  Unexpected whitespace before ")"   function-parentheses-space-inside
 (...)
 73:13  ✖  Unexpected whitespace after "("    function-parentheses-space-inside
 75:4   ✖  Unexpected whitespace before ")"   function-parentheses-space-inside
 ```
 I'm not sure why. when I remove the whitespaces the error transforms into:
 ```
  64:3   ✖  Expected indentation of 4 spaces   indentation                      
```
Could it be because I'm changing svg path info on these lines? they refer to:
```
    d: path(
      'M 24.94 35.61 L 38.27 22.28 L 36.38 20.39 L 24 32.78 L 18.28 27.06 L 16.39 28.95 L 23.06 35.62 C 23.31 35.86 23.65 36 24 36 C 24.35 36 24.69 35.86 24.94 35.61 Z'
    );
```

Reply:
@ethan605 made a go at the `animations.scss` file on [this PR](https://github.com/onfido/onfido-sdk-ui/pull/1431/commits/481fa8176ea881eacfc0410c5d7b677ca6837ac8) 😁